### PR TITLE
Update docs using xamarin/xamarin-android@ca8f5617

### PR DESCRIPTION
**Pull request by pjcollins**
Fixes the remaining `JavadocImport-Error` instances that would occur
when generating API docs for `Mono.Android.dll`.

Notable documentation related generator changes:

  * xamarin/java.interop@a0728ed5: [Java.Interop.Tools.JavaSource] Fix remaining parsing errors

Changes in xamarin-android and java.interop:

  * https://github.com/xamarin/xamarin-android/compare/ace5f71c40d05070612cb860f38764eae3deef59...ca8f5617cc55cc3510a61433d8d50c6a58ec0ac2
  * https://github.com/xamarin/java.interop/compare/032f1e7160c23a78e5b37208542f763e29992067...a0728ed5b4e2eb91c2c8048b70c221f19c4b6474

### CODE REVIEW

File: docs/Mono.Android/en/Android.Animation/FloatArrayEvaluator.xml [hunk 1]
```diff
@@ -31,14 +31,18 @@
     </Attribute>
   </Attributes>
   <Docs since="21">
-    <summary>This evaluator can be used to perform type interpolation between <c>float[]</c> values.</summary>
+    <summary>This evaluator can be used to perform type interpolation between &lt;code&gt;float[]&lt;/code&gt; values.</summary>
```
Comment (jonpryor) [inline docs/Mono.Android/en/Android.Animation/FloatArrayEvaluator.xml:34]: This change is not an improvement.

Comment (pjcollins) [reply docs/Mono.Android/en/Android.Animation/FloatArrayEvaluator.xml:34]: Unfortunately this is not a new issue, just a byproduct of the new javadoc tooling now being able to process this type. I've filed a bug for this - https://github.com/xamarin/java.interop/issues/1041

File: docs/Mono.Android/en/Android.App.Admin/DevicePolicyManager.xml [hunk 2]
```diff
@@ -7112,12 +7112,29 @@
...
-        <param name="agent">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="admin">Which <c>DeviceAdminReceiver</c> this request is associated with. If null,
+            this function returns a list of configurations for all admins that declare
+            <c>#KEYGUARD_DISABLE_TRUST_AGENTS</c>. If any admin declares
+            <c>#KEYGUARD_DISABLE_TRUST_AGENTS</c> but doesn't call
+            <c>#setTrustAgentConfiguration(ComponentName, ComponentName, PersistableBundle)</c>
+            for this {</param>
+        <param name="agent}}">or calls it with a null configuration, null is returned.</param>
```
Comment (pjcollins) [inline docs/Mono.Android/en/Android.App.Admin/DevicePolicyManager.xml:7121]: Filed an issue for this https://github.com/xamarin/java.interop/issues/1042, it's present in a few other places.

File: docs/Mono.Android/en/Android.Graphics/ColorSpace+Rgb.xml [hunk 3]
```diff
@@ -18,14 +18,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>{</summary>
```
Comment (pjcollins) [inline docs/Mono.Android/en/Android.Graphics/ColorSpace+Rgb.xml:21]: https://github.com/xamarin/java.interop/issues/1043

Comment (pjcollins) [reply docs/Mono.Android/en/Android.Graphics/ColorSpace+Rgb.xml:21]: Will be fixed in a future update.

File: docs/Mono.Android/en/Android.App.Admin/DevicePolicyManager.xml (hunk 2 above)
Comment (pjcollins) [reply docs/Mono.Android/en/Android.App.Admin/DevicePolicyManager.xml:7121]: Will be fixed in a future update.

**Pull request merged**